hlx_statics/blocks/announcement/announcement.js

```diff
@@ -71,9 +72,12 @@ export default async function decorate(block) {
     p.style.whiteSpace = "normal";
   });
   const imageExists = block.querySelector('picture img');
+  const allowed = ["background-color-white", "background-color-navy", "background-color-dark-gray"];
   if (!imageExists) {
-    if (!block.classList.contains('background-color-white') && !block.classList.contains('background-color-navy') && !block.classList.contains('background-color-dark-gray')) {
-      block.classList.add('background-color-gray');
+    if (getMetadata("template") === "documentation" && background_Exists) {
```

Comment:
Looking at previous code, DevDocs also restricted the allowed background colors, so we still need to keep that logic

```suggestion
    if (getMetadata("template") === "documentation" && Boolean(backgroundColor) && allowedBackgroundColors.includes(backgroundColor)) {
```